samples/bluetooth/central_esl/central_esl_subcommands.rst

```diff
@@ -0,0 +1,226 @@
+.. _central_esl_subcommands:
+
+Central ESL subcommands
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. _esl_c_subcmds:
+
+``esl_c`` is the ESL Access Point client shell command and it supports the following subcommands.
```

Comment:
Place these texts under the corresponding headings and add colon (:) instead of the full stop in the end. I believe this belongs under "ESL Client subcommands".